data/database.py

```diff
@@ -2048,6 +2048,32 @@ class ManifestSecurityStatus(BaseModel):
     metadata_json = JSONField(default={})
 
 
+class ProxyCacheConfig(BaseModel):
+    """
+    Represents the configuration for an organization of type proxy cache
+    """
+
+    user = ForeignKeyField(User)
+    creation_date = DateTimeField(default=datetime.utcnow)
+    upstream_registry = CharField(max_length=270)
```

Comment:
good question. here are some of the possible values this might take:
* `docker.io`
* `docker.io/library`
* `quay.io`
* `quay.io/projectquay`
* `my-registry.example`
I'm actually having a hard time chosing a better name. It's not a url, neither an endpoint, and definitely not the registry name.
the `registries.conf`[1] documentation calls this an image name `prefix`... which in is really what this is, but I'm not sure if referring to it like that would be interpreted in the same way by everyone reading the code.

Thoughts?

[1] https://github.com/containers/image/blob/main/docs/containers-registries.conf.5.md#choosing-a-registry-toml-table